packages/docs/src/locales/en/en.json

```diff
@@ -3513,7 +3522,76 @@
     "links": "Links",
     "textHighlighted": "Highlighted text",
     "blockquote": "Blockquote",
-    "textBlock": "Text block"
+    "textBlock": "Text block",
+    "examples": {
+      "blockQuote": {
+        "content": "BQ: Let’s talk about meat fondue recipes and what you need to know first. Meat fondue also known as oil fondue is a method of cooking all kinds.",
+        "author": "— Mister Lebowski"
+      },
+      "codeSnippet": {
+        "wonderfulExample": "This is a wonderful example.",
+        "readMore": "Read more"
+      },
+      "headings": {
+        "display1": "Display 1 Heading",
+        "display2": "Display 2 Heading",
+        "display3": "Display 3 Heading",
+        "display4": "Display 4 Heading",
+        "display5": "Display 5 Heading",
+        "display6": "Display 6 Heading"
+      },
+      "links": {
+        "defaultLink": "Default Link",
```

Comment:
🥲 
In these cases text is either keyword (from "Secondary Link" user can somewhat derive that class is `link-secondary`, which is not the case for translated text) or filler text.

So not applicable for translation.

I [updated docs](https://github.com/epicmaxco/vuestic-ui/pull/1580) to better represent this idea.